django/contrib/admin/templates/admin/change_form.html

```diff
@@ -73,15 +73,39 @@
             $(document).ready(function() {
```

Comment:
For line 82 / 105 depending on what column you are reading:
{% if adminform and add %}